### ISSUE TITLE
Upgrade npm packages

### ISSUE THREAD

**Issue by annez** [uswitch/ustyle#193]
Grunt is now at 1.0.0 and a lot of our packages are out of date. We should take a look and upgrade them.

- scss-lint
- stylestats
- cssstats
- jscs
- sassdoc

Are all important to our development and should be upgraded.

**Comment by jpstevens** [comment]
@annez RE "should be upgraded", can you give reasons?

**Comment by annez** [comment]
Yes, most of these packages were never in a stable release so they've been constantly upgraded and improved
- scss-lint has added more features for future CSS, plus plenty of bug fixes
- cssstats is giving out more detailed and easier to digest stats in the latest versions
- same applies to stylestats, both which we should be more prominently using to understand our CSS complexity and size (or not, depending on the performance argument nowadays)
- sassdoc has added some description features (nothing major) http://sassdoc.com/customising-the-view/#description

With grunt and the watch tasks - they no longer work as expected and it seems you need to save `.scss` files twice to get the right compilation. This is probably a dependency issue around Node 4+ and old version of file watcher and grunt. 

Get the error:
```
Fatal error: this.ee._events.hasOwnProperty is not a function
```

Every time we run the watcher task

![image](https://cloud.githubusercontent.com/assets/1567681/19854122/221d6622-9f65-11e6-9ee8-508f0bd812fd.png)

**Comment by marcosefrem** [comment]
Hi, and what can I do for solve the problem? I have updated the packages and it doesn't work

**Comment by annez** [comment]
Hi @marcosefrem - can you show us a diff of the packages you updated and a stack trace of what is erroring?

**Comment by marcosefrem** [comment]
Here you have, thanks in advance

![packages](https://cloud.githubusercontent.com/assets/22935202/25311867/c7e074a8-280b-11e7-9e13-611a36537ecc.png)
![error](https://cloud.githubusercontent.com/assets/22935202/25311868/c7ff0a26-280b-11e7-9400-00935f6249a0.png)
